src/main/java/com/icegreen/greenmail/imap/commands/SearchTermBuilder.java

```diff
@@ -299,7 +300,7 @@ private static SearchTermBuilder createDateSearchTermBuilder(final int compariso
             public SearchTerm build() {
                 String dateStr = getStringParameter(0);
                 try {
-                    Date date = new SimpleDateFormat("dd-MMM-yyyy").parse(dateStr);
+                    Date date = new SimpleDateFormat("dd-MMM-yyyy", Locale.ENGLISH).parse(dateStr);
```

Comment:
How does this work in French locale? Is the date format the same? What about other locales where date format is different (mm/dd/yy for example)?

Reply:
I believe it now sets `Locale.ENGLISH` as a base filter and parse it for `ReceivedDateTerm` or `SentDateTerm` methods.

Reply:
Re-wrote the Class/Method accordingly: https://github.com/Alfresco/alfresco-greenmail/blob/763e4ecab4ebc7ed21fce9bed3094cec6a816182/src/test/java/com/icegreen/greenmail/imap/commands/SearchTermBuilderTest.java#L33-L43